cdk/kittyhawk/API.md

```diff
@@ -38,14 +38,15 @@ You can find the full reference [here](https://kittyhawk.pennlabs.org/interfaces
 DjangoApplication and ReactApplication have special properties that override properties from Application.
 
 For DjangoApplication:
-- domain (string) - Domain for the application. (**Required**)
+- domains (Array\<String, Boolean\>) - Domains for the application, and whether each domain should be treated as a subdomain. (**Required**)
 - ingressPaths (Array\<String\>) - List of paths that should be available on the domain. (**Required**)
 - djangoSettingsModule (string) - DJANGO_SETTINGS_MODULE environment variable. (**Required**)
 
 For ReactApplication:
 - domain (string) - Domain for the application. (**Required**)
 - ingressPaths (Array\<String\>) - List of paths that should be available on the domain. (**Required**)
 - portEnv (string) - PORT environment variable. (**Optional**, default '80')
+- isSubdomain (boolean) - If the domain should be treated as a subdomain for certificate purposes.
```

Comment:
Nit: I feel like this should be an optional boolean, and by default it is just false. We have to specify `isSubdomain: false` in a lot of areas for normal domains.

Reply:
addressed